dataland-backend/src/main/kotlin/org/dataland/datalandbackend/services/DataManager.kt

```diff
@@ -52,31 +38,11 @@ class DataManager(
     @Autowired private val metaDataManager: DataMetaInformationManager,
     @Autowired private val storageClient: StorageControllerApi,
     @Autowired private val cloudEventMessageHandler: CloudEventMessageHandler,
-    @Autowired private val messageUtils: MessageQueueUtils,
+    @Autowired private val dataManagerUtils: DataManagerUtils,
 ) {
     private val logger = LoggerFactory.getLogger(javaClass)
     private val publicDataInMemoryStorage = mutableMapOf<String, String>()
 
-    private fun assertActualAndExpectedDataTypeForIdMatch(
-        dataId: String,
-        dataType: DataType,
-        correlationId: String,
-    ) {
-        val dataMetaInformation = metaDataManager.getDataMetaInformationByDataId(dataId)
-        if (DataType.valueOf(dataMetaInformation.dataType) != dataType) {
-            throw InvalidInputApiException(
-                "Requested data $dataId not of type $dataType",
-                "The data with the id: $dataId is registered as type" +
-                    " ${dataMetaInformation.dataType} by Dataland instead of your requested" +
-                    " type $dataType.",
-            )
-        }
-        logger.info(
-            "Requesting Data with ID $dataId and expected type $dataType from framework data storage. " +
-                "Correlation ID: $correlationId",
-        )
-    }
-
     /**
      * Method to make the data manager add data to a data store, store metadata in Dataland and sending messages to the
      * relevant message queues
```

Comment:
Statt `storeDataSetInMemoryAndSendReceptionMessageAndPersistMetaInfo()` vllt besser einfach `processDataStorageRequest()`
Ähnlich wie im PrivateDataManager

Reply:
Kann ich ändern, die Sachen sind im Prinzip aber so wie sie in main waren.

Reply:
Geändert